src/bloom/BloomFilter.ts

```diff
@@ -139,5 +183,68 @@ export class BloomFilter extends AlphaFilter
         this._blurXFilter.blur = this.strengthX;
         this._blurYFilter.blur = this.strengthY;
     }
+
+    /**
+     * @deprecated since 8.0.0
+     *
+     * The strength of both the blurX and blurY properties simultaneously
+     * @default 2
+     */
+    get blur(): number
+    {
+        deprecation(v8_0_0, 'BloomFilter.blur is deprecated, please use BloomFilter.strength instead');
+
+        return this.strengthX;
+    }
+    set blur(value: number)
+    {
+        deprecation(v8_0_0, 'BloomFilter.blur is deprecated, please use BloomFilter.strength instead');
+
+        this.strength = value;
+    }
+
+    /**
+     * @deprecated since 8.0.0
+     *
+     * The strength of the blurX property
+     * @default 2
+     */
+    get blurX(): number
+    {
+        deprecation(v8_0_0, 'BloomFilter.blurX is deprecated, please use BloomFilter.strengthX instead');
+
+        return this.strengthX;
+    }
+    set blurX(value: number)
+    {
+        deprecation(v8_0_0, 'BloomFilter.blurX is deprecated, please use BloomFilter.strengthX instead');
+
+        this.strengthX = value;
+    }
+
+    /**
+     * @deprecated since 8.0.0
+     *
+     * The strength of the blurY property
+     * @default 2
+     */
+    get blurY(): number
+    {
+        deprecation(v8_0_0, 'BloomFilter.blurY is deprecated, please use BloomFilter.strengthY instead');
+
+        return this.strengthY;
+    }
+    set blurY(value: number)
+    {
+        deprecation(v8_0_0, 'BloomFilter.blurY is deprecated, please use BloomFilter.strengthY instead');
+
+        this.strengthY = value;
+    }
 }
 
+function convertDeprecatedBlurValue(value: DeprecatedBlurValue): PointData | number
+{
+    if (typeof value === 'number' || ('x' in value && 'y' in value)) return value;
+
+    return { x: value[0], y: value[1] };
+}
```

Comment:
I think this can be inlined in the constructor. We don't need an extra function call and we're already doing the conditional check there.